ui/components/DashboardComponent/resources/network/config.js

```diff
@@ -169,9 +174,14 @@ export const NetWorkTableConfig = (switchView, meshSyncResources, k8sConfig) =>
             sortThirdClickReset: true,
             customBodyRender: function CustomBody(val) {
               let clusterName = getClusterNameFromClusterId(val, k8sConfig);
+              let connectionId = getConnectionIdFromClusterId(val, k8sConfig);
               return (
                 <>
-                  <ConnectionChip title={clusterName} iconSrc="/static/img/kubernetes.svg" />
+                  <ConnectionChip
+                    title={clusterName}
+                    iconSrc="/static/img/kubernetes.svg"
```

Comment:
This should be coming from the ConnectionDefinition, @sudhanshutech

Reply:
okay, Uzair explained this structure. Moving now with this pr for the ad-hoc, the icon will set to dynamic in next pr